easybuild/easyconfigs/r/reframe/reframe-3.0-dev6.eb

```diff
@@ -0,0 +1,52 @@
+# @author: gppezzi, vkarak, manitart
+
+easyblock = 'Tarball'
+
+name = 'reframe'
+version = '3.0-dev6'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = 'A regression framework for writing portable tests for HPC systems.'
+
+toolchain = SYSTEM
+
+sources = [
+    {
+        'filename': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/eth-cscs/reframe/archive/']
+    },
+    {
+        'filename': '/apps/common/easybuild/sources/%(nameletterlower)s/'
+                    '%(name)s/reframe-tests-%(version)s.tar.gz'
+    }
+]
+
+builddependencies = [('pip', '20.0.2', '-py3')]
+
+keepsymlinks = True
+postinstallcmds = [
+    'cp %(installdir)s/config/cscs.py %(installdir)s/reframe/settings.py',
+    'cp -r %(builddir)s/reframe-tests-%(version)s/checks %(installdir)s/cscs-checks/private',
+    'mv %(installdir)s/cscs-checks %(installdir)s/checks',
+    'PYTHONUSERBASE=%(installdir)s/installed_by_pip pip install --upgrade '
+    '--user -r %(builddir)s/reframe-%(version)s/requirements.txt pygelf',
+    'python3 -m venv docs_env && source docs_env/bin/activate && '
+    'pip install -r docs/requirements.txt && make -C docs man && deactivate && '
+    'mkdir -p %(installdir)s/share/man/man1 %(installdir)s/share/man/man8 && '
+    'cp docs/man/reframe.1 %(installdir)s/share/man/man1 && '
+    'cp docs/man/reframe.settings.8 %(installdir)s/share/man/man8'
```

Comment:
Is there a way that this can done also from pip? Because then the pip installation diverges. Also the tutorial should also be installed by the pip installation.

Reply:
Then the `setup.py` script of reframe should be changed to include the manual as well as the tutorial.

Reply:
I will open an issue in the reframe repo.